builder.go

```diff
@@ -488,7 +488,7 @@ func (r *DB) LockForUpdate() *DB {
 // Dump prints raw sql to stdout
 func (r *DB) Dump() {
 	log.SetOutput(os.Stdout)
-	log.Println(r.Builder.buildSelect())
+	// log.Println(r.Builder.buildSelect())
```

Comment:
What`s this comment for? Seems like duplicate

Reply:
My bad. Updated ~~forced~~ the commit.